wordle-web/components/navigation-dialog.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div>
+    <v-btn icon @click="toggleDialog">
+      <v-icon> mdi-hamburger</v-icon>
+    </v-btn>
+
+    <v-dialog v-model="dialog" width="450">
+      <v-card>
+        <v-container>
+          <v-card-title> Navigation</v-card-title>
+
+          <v-card-text>
+            <v-menu offset-y>
+              <template #activator="{ on, attrs }">
+                <v-list dark v-bind="attrs" v-on="on">
+                  <v-list-item-group>
+                    <v-list-item nuxt to="/" @click="toggleDialog">
+                      <v-btn color="primary"> Home </v-btn>
+                    </v-list-item>
+                    <v-list-item nuxt to="/game" @click="toggleDialog">
+                      <v-btn color="primary"> Game </v-btn>
+                    </v-list-item>
+                    <v-list-item nuxt to="/about" @click="toggleDialog">
+                      <v-btn color="primary"> About </v-btn>
+                    </v-list-item>
+                  </v-list-item-group>
+                </v-list>
+              </template>
+            </v-menu>
```

Comment:
Typically you wouldn't put buttons inside of a list like this.
```suggestion
            <v-menu offset-y>
              <template #activator="{ on, attrs }">
                <v-list dark v-bind="attrs" v-on="on">
                  <v-list-item-group>
                    <v-list-item nuxt to="/" @click="toggleDialog">
                      <v-list-item-title> Home </v-list-item-title>
                    </v-list-item>
                    <v-list-item nuxt to="/game" @click="toggleDialog">
                      <v-list-item-title> Game </v-list-item-title>
                    </v-list-item>
                    <v-list-item nuxt to="/about" @click="toggleDialog">
                      <v-list-item-title> About </v-list-item-title>
                    </v-list-item>
                  </v-list-item-group>
                </v-list>
              </template>
            </v-menu>
```